src/makeCalculator.js

```diff
@@ -36,8 +36,50 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+    methods: ['add', 'subtract', 'multiply', 'divide', 'reset', 'operate'],
+
+    operate(method, arg) {
+      method.call(this, arg);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(arg) {
+      this.result += arg;
+
+      return this.result;
```

Comment:
return this should be only for operate and reset methods, please check again the task description
<img width="969" alt="Screenshot 2023-11-06 at 18 54 24" src="https://github.com/mate-academy/js_advanced_calculator/assets/64746943/af7246f1-02b5-4dff-96f7-1f9a0404eac1">
```suggestion
```